proposed/http-message.md

```diff
@@ -1133,8 +1141,8 @@ interface StreamableInterface
      * @param int $length Read up to $length bytes from the object and return
      *     them. Fewer than $length bytes may be returned if underlying stream
      *     call returns fewer bytes.
-     * @return string|false Returns the data read from the stream, false if
-     *     unable to read or if an error occurs.
+     * @return string Returns the data read from the stream
+     * @throws \RuntimeException if unable to read or if an error occurs.
```

Comment:
In the case of this:

>  If the object is in non-blocking mode and no bytes > are available, None is returned.

what would be the PHP equivalent of Python's `None`? `null`?